src/backend/defs/rest-api/objects/Attachment.yaml

```diff
@@ -1,13 +1,11 @@
 ---
 type: object
 properties:
-  attachment_id:
```

Comment:
hmm does it mean that we only use an index ?
What about this use case : 

1. clic DELETE attachments/0
2. clic DELETE attachments/1
3. DELETE 1 success
4. invalidate message
5. -> DELETE 2 will probably fail